src/schema/v2/schema.ts

```diff
@@ -292,25 +297,33 @@ export default new GraphQLSchema({
   mutation: new GraphQLObjectType<any, ResolverContext>({
     name: "Mutation",
     fields: {
+      addOrderedSetItem: addOrderedSetItemMutation,
```

Comment:
#minor i wonder if the naming on these should be `addItemToSet` or something. That feels generic-y enough to represent it accurately, vs. `addOrderedSetItem` which feels a bit...implementation-leak-y? like what's an 'ordered set item'? etc. #pedantic

Reply:
Yeah, good call on this. We're a little inconsistent with the naming; sometimes using `set` and sometimes `OrderedSet`. How about if we just drop `Ordered` and stick with using set?

Reply:
Went to change this, as I agree it is implementation-leaky-y, but ultimately decided to favor consistency of naming here. We're already using ordered sets, and ordered items in the queries, and think it is slightly better to be consistent in this case.